python/pyspark/ml/feature.py

```diff
@@ -2245,6 +2248,21 @@ def getMinCount(self):
         """
         return self.getOrDefault(self.minCount)
 
+    @since("2.0.0")
+    def setWindowSize(self, value):
+        """
+        Sets the value of :py:attr:`windowSize`.
+        """
+        self._paramMap[self.windowSize] = value
```

Comment:
Rather than directly accessing the param map, use the `_set` function (see https://github.com/apache/spark/pull/11939 / SPARK-14104)